configuration/management/commands/refresh_views.py

```
@@ -0,0 +1,91 @@
+                try:
+                    msg = f"Refreshing {view}... {self.style.SUCCESS('Done')}"
+                    ident = connection.ops.quote_name(view)  # safe for unqualified names
+                    cursor.execute(f"REFRESH MATERIALIZED VIEW {ident};")
+                    self.stdout.write(msg)
+                except DatabaseError as e:
+                    db_errors.append((view, str(e)))
+                except ProgrammingError as e:
+                    skipped.append((view, str(e)))
+                    msg = f"Refreshing {view}... {self.style.WARNING('Skipped')}"
+                    self.stdout.write(msg)
+                except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)